src/lib/support/Span.h

```diff
@@ -287,7 +330,7 @@ using MutableCharSpan = Span<char>;
 
 inline CHIP_ERROR CopySpanToMutableSpan(ByteSpan span_to_copy, MutableByteSpan & out_buf)
 {
-    VerifyOrReturnError(IsSpanUsable(span_to_copy), CHIP_ERROR_INVALID_ARGUMENT);
+    VerifyOrReturnError(!span_to_copy.empty(), CHIP_ERROR_INVALID_ARGUMENT); // why?
```

Comment:
This was not about empty, it was related to nullptr. If nullptr is no longer valid, then this is not needed. SpanUsable related to avoiding trying to copy to/from nullptr.